app/views/reports/elements/_my_module_repository_element.html.erb

```diff
@@ -1,6 +1,7 @@
 <% my_module ||= @my_module %>
-<% element_id ||= repository&.id %>
+<% repository ||= nil %>
```

Comment:
this doesn't look like it does anything, as it will assign `nil` if it's already `nil`